firmware/user/hpatimer.c

```diff
@@ -198,6 +199,7 @@ void ICACHE_FLASH_ATTR setBuzzerNote(notePeriod_t note)
  */
 void ICACHE_FLASH_ATTR startBuzzerSong(const song_t* song)
 {
+    os_printf("startBuzzerSong: %p\n",song);
```

Comment:
Make sure this debugging is removed / commented before merging

Reply:
Reverted this commit.